packages/runtime-handler/src/dev-runtime/types.ts

```diff
@@ -0,0 +1,88 @@
+export type SearchConfig = {
```

Comment:
`runtime-handler` is now exporting some duplicate types that also live in `serverless-api` or `twilio-run`. Should they be consolidated somewhere? 

Also in `serverless-api`:
`SearchConfig`, `AccessOptions`, `ServerlessResourceConfig`, `ServerlessResourceConfigWithFilePath`

Also in `twilio-run`:
`EnvironmentVariablesWithAuth`

Reply:
The problem is the way the dependency chain works. `twilio-run` should rely on `serverless-api` and `runtime-handler` (at least during compile time) but `runtime-handler` shouldn't rely on `twilio-run`. And similarly I didn't want `runtime-handler`to rely on `serverless-api` since `runtime-handler` can be different from user to user during runtime so it should be self-contained in general

Reply:
In fact you'll see that runtime-handler is only a devDependency in twilio-run because I do `import type` as opposed to `import`.

Reply:
Could runtime-handler take serverless-api as a devDependency and do `import type` too then?

Reply:
Unfortunately not because some of these are part of `ServerConfig` which is exported by `runtime-handler`. Realistically the only thing we could do is to move it into a common dependency that all of them consume. But realistically they are all part of `ServerConfig` which is basically the "contract" for runtime-handler. So I could see scenarios where they could diverge slightly from the other use cases.

I was able to remove `SearchConfig` though since it's not being used anymore.